addons/website/controllers/main.py

```diff
@@ -405,6 +405,31 @@ def autocomplete(self, search_type=None, term=None, order=None, limit=5, max_nb_
             results_data.sort(key=lambda r: r.get('name', ''), reverse='name desc' in order)
         results_data = results_data[:limit]
         result = []
+
+        def get_mapping_value(field_type, value):
+            if field_type == 'text':
+                if value:
+                    value = shorten(value, max_nb_chars, placeholder='...')
+                if field_meta.get('match') and value and term:
```

Comment:
`field_meta` is only defined in the loop that calls this function.
What about passing it as a parameter to help readability ?

Reply:
Owh, yes, you're right ! I thought I added it, seems like I thrown this parameter away at a time, don't know why.